packages/cdk8s/src/app.ts

```diff
@@ -9,11 +9,20 @@ import { Names } from './names';
 
 export interface AppOptions {
   /**
-   * The directory to output Kubernetes manifests.
+   * The directory to output Kubernetes manifests. Mutually exclusive with
+   * the `--stdout` option.
    *
    * @default - CDK8S_OUTDIR if defined, otherwise "dist"
    */
   readonly outdir?: string;
+
+  /**
+   * Whether or not to print to stdout. Mutually exclusive with
+   * the `--outdir` option.
+   *
+   * @default - true if defined, false otherwise
+   */
+  readonly stdout?: boolean;
```

Comment:
mmmm.... not sure I am super comfortable with this implemented here. I am wondering if we should just implement this in the CLI by emitting the files to a temporary directory and then just `cat`ting those files to STDOUT. I don't know that this is something we really need in the framework. @iliapolo what do you think?

Reply:
Yeah, I can move it to the CLI like that, works for me!